tools/NuGet/template-artifactory/DynamoVisualProgramming.CoreRuntimeNet6.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+    <metadata>
+        <id>DynamoVisualProgramming.CoreRuntimeNet6</id>
```

Comment:
@sm6srw any reason for the new naming? I was thinking of keeping the name and just append framework name to be `DynamoCoreRuntimeNET60`

Reply:
Not really. I am usually really bad at picking new names as you can see. I like your reasoning about the name better so please go ahead and rename it. We need to ping D4R and C3D about the change. Better to do it know than later.